src/nvim/window.c

```diff
@@ -803,6 +803,7 @@ void ui_ext_win_position(win_T *wp, bool validate)
   if (!wp->w_floating) {
     ui_call_win_pos(wp->w_grid_alloc.handle, wp->handle, wp->w_winrow,
                     wp->w_wincol, wp->w_width, wp->w_height);
+    ui_put_grid(&wp->w_grid_alloc, false);
```

Comment:
I don't think this should be done. in multigrid mode, there is no guarantee that the grid size of the specific windows will match the space in default_grid (that's the entire point of ext_multigrid mode), so things will get weird in other ways. Also it breaks an abstraction boundary to ui_compositor.c which is not something I like for a edge case unrelated to ordinary use.